pyccel/codegen/printing/fcode.py

```diff
@@ -2651,6 +2653,8 @@ def _print_NumpyUfuncBase(self, expr):
             func_name = numpy_ufunc_to_fortran[type_name]
         except KeyError:
             self._print_not_supported(expr)
+        if func_name.startswith('numpy_'):
+            self._additional_imports.add(Import('numpy_f90', Module('numpy_f90',(),())))
         args = [self._print(NumpyFloat(a) if a.dtype is NativeInteger() else a)\
 				for a in expr.args]
```

Comment:
See above. Would a `_print_NumpySign` function be useful?

Reply:
Do you suggest to put back the `_print_NumpySign`? Definitely it was working fine, but it seems like it wasn't readable.

Reply:
I don't remember what was unreadable. Was it this PR that used `x` but didn't have it saved in the `_attribute_nodes` ? I don't think it was unreadable, just possibly unnecessary as all it did was duplicate the code in `_print_NumpyUfuncBase`. However if different code is needed to solve the printing problem then a `_print_NumpySign` function becomes useful again

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/32e8a921d1a7389b253fb12f7cd4c4c3a5445182 I kept the fortran module and added separate printer